metabase/metabase
# Add e2e tests for local search in the entity picker

Epic https://github.com/metabase/metabase/issues/47281
Testing plan https://github.com/metabase/metabase/issues/47282

Adds tests for all cases from the testing plan. Fixes found bugs.

How to verify:
- New -> Question
- "Tables" -> Enter search text -> See that there are 2 tabs, with your database being selected by default
- "Saved questions" -> Enter search text -> See that there are 2 tabs, with your collection being selected by default
